CRM/Api4/Services.php

```diff
@@ -86,7 +86,7 @@ public static function loadServices($namespace, $tag, $container) {
         $container->addResource($resource);
         foreach (glob("$path*.php") as $file) {
           $matches = [];
-          preg_match('/(\w*).php/', $file, $matches);
+          preg_match('/(\w*)\.php$/', $file, $matches);
```

Comment:
I haven't tested this scenario, but this does look like a more correct implementation of "find things ending in `*.php`" :+1:

Reply:
@totten i'm guessing also if there was a problem with this right then the api_v4_unit tests would fail right because this merges in the core resources + extension resources